code-of-conduct.md

```diff
@@ -0,0 +1,46 @@
+census-developers@googlegroups.com
```

Comment:
what does this top line with an e-mail represent?

Reply:
I think it accidentally got in there. Will be removing it.